db/migrate/20240918145749_change_from_and_to_participant_ids_to_uuid_in_participant_id_changes.rb

```diff
@@ -0,0 +1,8 @@
+class ChangeFromAndToParticipantIdsToUuidInParticipantIdChanges < ActiveRecord::Migration[7.1]
+  def up
+    change_column :participant_id_changes, :from_participant_id, :uuid
```

Comment:
discussed in person: there is a failure on deploy:
```
ActiveRecord::StatementInvalid: PG::DatatypeMismatch: ERROR:  column "from_participant_id" cannot be cast automatically to type uuid
```
we need to revisit this migration :)

Reply:
Should we assume this table doesn't already exist and we add data with migration only?
Otherwise, we'll have to have a step to migrate existing data to `ecf_id`.

Locally we'll have to reset db.

Reply:
Yeah ideally we should need to, if we had existings records in the table.

- Add a new uuid field
- Run a update to populate the new uuid
- Drop the old bigint ID field
- Rename the new uuid field to the old name

As we don't have existing records in the DB, we should be fine with this "quickfix".